packages/@o3r/rules-engine/schematics/ng-update/v11.6/use-register-action-handlers.ts

```diff
@@ -0,0 +1,20 @@
+import type { Rule, Tree } from '@angular-devkit/schematics';
+import { findFilesInTree } from '@o3r/schematics';
+
+/**
+ * Replace `actionHandlers.add` or `actionHandlers.delete` by `registerActionHandlers` or `unregisterActionHandlers` in file
+ * @param tree
+ */
+export const useRegisterActionHandlers: Rule = (tree: Tree) => {
+  const files = findFilesInTree(tree.root, (file) => /actionHandlers\.(add|delete)/.test(tree.readText(file)));
```

Comment:
Shouldn't we be more restrictive on the regex to avoid impacting unrelated code?
```suggestion
  const files = findFilesInTree(tree.root, (file) => /\.actionHandlers\.(add|delete)/.test(tree.readText(file)));
```